spanner/api/Spanner/Program.cs

```diff
@@ -1343,6 +1378,177 @@ await connection.CreateDdlCommand($"DROP TABLE {table}")
+                + $"/databases/{databaseId}";
+
+            // Get current max SingerId.
+            Int64 maxSingerId = 0;
+            using (var connection = new SpannerConnection(connectionString))
+            {
+                // Execute a SQL statement to get current MAX() of SingerId.
+                var cmd = connection.CreateSelectCommand(
+                    @"SELECT MAX(SingerId) as SingerId FROM Singers");
+                using (var reader = await cmd.ExecuteReaderAsync())
+                {
+                    while (await reader.ReadAsync())
+                    {
+                        long parsedValue;
+                        bool result = Int64.TryParse(
+                            reader.GetFieldValue<string>("SingerId"), out parsedValue);
+                        if (result) maxSingerId = parsedValue;
+                    }
+                }
+            }
+
+            // Batch insert 249,900 singer records into the Singers table.
+            using (var connection = new SpannerConnection(connectionString))
+            {
+                await connection.OpenAsync();
+
+                for (int i = 0; i < 100; i++)
+                {
+                    using (var tx = await connection.BeginTransactionAsync())
+                    using (var cmd = connection.CreateInsertCommand("Singers", new SpannerParameterCollection
+                    {
+                        {"SingerId", SpannerDbType.String},
+                        {"FirstName", SpannerDbType.String},
+                        {"LastName", SpannerDbType.String}
+                    }))
+                    {
+                        cmd.Transaction = tx;
+                        for (var x = 1; x < 2500; x++)
+                        {
+                            maxSingerId++;
+                            string nameSuffix = Guid.NewGuid().ToString().Substring(0, 8);
+                            cmd.Parameters["SingerId"].Value = maxSingerId;
+                            cmd.Parameters["FirstName"].Value = $"FirstName-{nameSuffix}";
+                            cmd.Parameters["LastName"].Value = $"LastName-{nameSuffix}";
+                            cmd.ExecuteNonQuery();
+                        }
+                        await tx.CommitAsync();
+                    }
+                }
+            }
+            Console.WriteLine("Done inserting sample records...");
+        }
+
+        // [START spanner_batch_client]
+        public static object BatchReadRecords(string projectId,
+             string instanceId, string databaseId)
+        {
+            var response =
+                DistributedReadAsync(projectId, instanceId, databaseId);
+            Console.WriteLine("Waiting for operation to complete...");
+            response.Wait();
+            Console.WriteLine($"Operation status: {response.Status}");
+            return ExitCode.Success;
+        }
+
+        private static async Task DistributedReadAsync(string projectId,

```

Comment:
Please, document all relevant methods with xmldoc.

Reply:
Opting not to do this to maintain the existing format of the code which does not include xmldoc comments for methods. We can revisit this for future code changes.